### ISSUE TITLE
tokens() panics on multibyte characters

### ISSUE THREAD

**Issue by fergusq** [ronjakoi/voikko-rs#3]
It seems that the `tokens` method cannot handle multibyte characters such as `ä` and `ö`.

The following code:

```rust
use voikko_rs::voikko;
  
fn main() {
    let v = voikko::Voikko::new("fi-x-morpho", None).unwrap();
    println!("{:?}", v.tokens("ä"));
}
```

Crashes with the following stacktrace:

```
thread 'main' panicked at 'byte index 1 is not a char boundary; it is inside 'ä' (bytes 0..2) of `ä`', src/libcore/str/mod.rs:2109:5
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:71
   2: std::panicking::default_hook::{{closure}}
             at src/libstd/sys_common/backtrace.rs:59
             at src/libstd/panicking.rs:211
   3: std::panicking::default_hook
             at src/libstd/panicking.rs:227
   4: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:491
   5: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:398
   6: rust_begin_unwind
             at src/libstd/panicking.rs:325
   7: core::panicking::panic_fmt
             at src/libcore/panicking.rs:95
   8: core::str::slice_error_fail
             at src/libcore/str/mod.rs:0
   9: core::str::traits::<impl core::slice::SliceIndex<str> for core::ops::range::Range<usize>>::index::{{closure}}
             at /rustc/9fda7c2237db910e41d6a712e9a2139b352e558b/src/libcore/str/mod.rs:1892
  10: <core::option::Option<T>>::unwrap_or_else
             at /rustc/9fda7c2237db910e41d6a712e9a2139b352e558b/src/libcore/option.rs:396
  11: core::str::traits::<impl core::slice::SliceIndex<str> for core::ops::range::Range<usize>>::index
             at /rustc/9fda7c2237db910e41d6a712e9a2139b352e558b/src/libcore/str/mod.rs:1892
  12: core::str::traits::<impl core::ops::index::Index<core::ops::range::Range<usize>> for str>::index
             at /rustc/9fda7c2237db910e41d6a712e9a2139b352e558b/src/libcore/str/mod.rs:1668
  13: voikko_rs::voikko::Voikko::tokens
             at /home/iikka/.cargo/registry/src/github.com-1ecc6299db9ec823/voikko-rs-1.0.3/src/lib.rs:367
  14: voikkorsbugi::main
             at src/main.rs:5
  15: std::rt::lang_start::{{closure}}
             at /rustc/9fda7c2237db910e41d6a712e9a2139b352e558b/src/libstd/rt.rs:74
  16: std::panicking::try::do_call
             at src/libstd/rt.rs:59
             at src/libstd/panicking.rs:310
  17: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:102
  18: std::rt::lang_start_internal
             at src/libstd/panicking.rs:289
             at src/libstd/panic.rs:398
             at src/libstd/rt.rs:58
  19: std::rt::lang_start
             at /rustc/9fda7c2237db910e41d6a712e9a2139b352e558b/src/libstd/rt.rs:74
  20: main
  21: __libc_start_main
  22: _start
```

**Comment by ronjakoi** [comment]
Sorry I haven't replied to this sooner. I didn't expect anyone to actually use this and apparently Github hasn't emailed me about new issues. I will take a look at this soon!

**Comment by ronjakoi** [comment]
Fixed in version 1.0.4. Please confirm? :)

**Comment by fergusq** [comment]
It works now :)
